tests/distributed/test_workspaces/envvars_ws/flow.yml

```diff
@@ -1,7 +1,7 @@
 jtype: Flow
 with:
   protocol: http
-  port_expose: ${{PORT_EXPOSE}}
+  port_expose: $PORT_EXPOSE
```

Comment:
This test used `${{PORT_EXPOSE}}`  because we support this syntax and we want to validate if it works.

Reply:
OK, I'll revert.